javascript/hangman.js

```diff
@@ -49,7 +80,9 @@ if (startGameButton) {
   });
 }
 
-document.addEventListener('keydown', event => {
+document.addEventListener("keydown", event => {
+  // if (checkIfLetter(event.keyCode) == true) {
+  // }
```

Comment:
you are missing the canvas things, but i think you see henriks code how he did it so you can follow it

Reply:
Ok, thanks :)